src/utils.jl

```diff
@@ -27,11 +27,5 @@ end
 
 _sturges(x) = StatsBase.sturges(length(x))
 
-@inline function _is_uniform_bins(A::AbstractVector{T}) where T<:Real
-    diffs = diff(A)
-    diff1 = first(diffs)
-    all(isapprox.(diff1, diffs; atol = 1e-9)) #hack
-end
-function _is_uniform_bins(A::AbstractRange{T}) where T<:Real
-    true
-end
+_is_uniform_bins(v::AbstractVector{<:Real}) = range(first(v), last(v), length=length(v)) == v
```

Comment:
floating point error may be issue here

Reply:
Ah ok I thought we wanted to make sure only if the user did `v=collect(1:0.1:5)` that it would still register as uniform binning. Anyway let me just scrap the PR because it decreases coverage and we'll keep using approx.

Reply:
while `l:s:h == collect(l:s:h)` will always be true, this may not:
```julia
julia> _s = (88000 - 36400) / 42
1228.5714285714287

julia> 36400:_s:88000 == range(36400, 88000; length=42)
false
```

Reply:
:( ok